experiments/need_shield_study.md

```diff
@@ -1,6 +1,72 @@
-# Experimentation using Shield (Studies)
+# I Need a Shield Study
```

Comment:
A lot of the contents of this document seem like they would be better placed in a higher level FAQ about experiments more broadly -- not about Shield studies.